plugins/GSdx/Renderers/Vulkan/GSDeviceVK.cpp

```diff
@@ -0,0 +1,238 @@
+/*
+ *	Copyright (C) 2019 Hamish Elliott
```

Comment:
`Copyright (C) 2019  PCSX2 Dev Team`
Not to take your credit but this is how we do it now.